projects/schematics/src/migrations/5_0/constructor-deprecations/data/cart-lib.migration.ts

```diff
@@ -0,0 +1,2253 @@
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'ActiveCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+    ],
+  },
+  {
+    class: 'CheckoutProgressMobileTopComponent',
+    importPath: '@spartacus/checkout/components',
+    deprecatedParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'CheckoutStepService',
+        importPath: '@spartacus/checkout/components',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'ActiveCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+    ],
+  },
+  {
+    class: 'NotCheckoutAuthGuard',
+    importPath: '@spartacus/checkout/components',
+    deprecatedParams: [
+      {
+        className: 'AuthService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'SemanticPathService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'Router',
+        importPath: '@angular/router',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'ActiveCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+    ],
+  },
+  {
+    class: 'PaymentMethodComponent',
+    importPath: '@spartacus/checkout/components',
+    deprecatedParams: [
+      {
+        className: 'UserPaymentService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'CheckoutFacade',
+        importPath: '@spartacus/checkout/root',
+      },
+      {
+        className: 'CheckoutDeliveryFacade',
+        importPath: '@spartacus/checkout/root',
+      },
+      {
+        className: 'CheckoutPaymentFacade',
+        importPath: '@spartacus/checkout/root',
+      },
+      {
+        className: 'GlobalMessageService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'ActivatedRoute',
+        importPath: '@angular/router',
+      },
+      {
+        className: 'TranslationService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'CheckoutStepService',
+        importPath: '@spartacus/checkout/components',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'ActiveCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+    ],
+  },
+  {
+    class: 'ReviewSubmitComponent',
```

Comment:
this one is also moved to `base`, renamed, and need manually check its constructor.